Source/santa/SNTAppDelegate.m

```diff
@@ -34,6 +37,15 @@ @implementation SNTAppDelegate
 #pragma mark App Delegate methods
 
 - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
+  if (@available(macOS 10.15, *)) {
+    LOGI(@"Requesting SystemExtension activation");
+    OSSystemExtensionRequest *req =
+        [OSSystemExtensionRequest activationRequestForExtension:@"com.google.santa.daemon"
+                                                          queue:dispatch_get_main_queue()];
```

Comment:
Does this block the thread?

Reply:
It did, yep. Much better on a non-main thread.